Log file created at: 2022/03/15 00:36:54
Running on machine: bf610164ce1a
Running duration (h:mm:ss): 0:00:00
Log line format: [IWEF]yyyymmdd hh:mm:ss.uuuuuu threadid file:line] msg
I20220315 00:36:54.197258     1 GraphDaemon.cpp:129] Starting Graph HTTP Service
I20220315 00:36:54.200800     7 WebService.cpp:123] Web service started on HTTP[19669]
I20220315 00:36:54.200878     1 GraphDaemon.cpp:143] Number of networking IO threads: 4
I20220315 00:36:54.200913     1 GraphDaemon.cpp:152] Number of worker threads: 4
I20220315 00:36:54.203145     1 MetaClient.cpp:82] Create meta client to "metad0":9559
I20220315 00:36:54.203183     1 MetaClient.cpp:83] root path: /usr/local/nebula, data path size: 0
I20220315 00:36:54.214560    31 ThriftClientManager-inl.h:67] resolve "metad1":9559 as "192.168.80.4":9559
I20220315 00:36:55.218748    31 ThriftClientManager-inl.h:67] resolve "metad2":9559 as "192.168.80.2":9559
I20220315 00:36:55.224016    32 ThriftClientManager-inl.h:67] resolve "metad2":9559 as "192.168.80.2":9559
I20220315 00:36:55.227876    33 ThriftClientManager-inl.h:67] resolve "metad2":9559 as "192.168.80.2":9559
I20220315 00:36:55.230438    34 ThriftClientManager-inl.h:67] resolve "metad2":9559 as "192.168.80.2":9559
I20220315 00:36:55.235473     1 MetaClient.cpp:3045] Load leader of "storaged0":9779 in 0 space
I20220315 00:36:55.235558     1 MetaClient.cpp:3045] Load leader of "storaged1":9779 in 0 space
I20220315 00:36:55.235584     1 MetaClient.cpp:3045] Load leader of "storaged2":9779 in 0 space
I20220315 00:36:55.235605     1 MetaClient.cpp:3051] Load leader ok
I20220315 00:36:55.237926     1 MetaClient.cpp:141] Register time task for heartbeat!
I20220315 00:36:55.266539     1 Snowflake.cpp:16] WorkerId init success: 1
I20220315 00:36:55.266952    38 GraphServer.cpp:58] Starting nebula-graphd on graphd:9669
I20220315 00:37:23.279208    22 GraphService.cpp:68] Authenticating user root from 192.168.80.11:49468
I20220315 00:37:48.285342    20 GraphService.cpp:68] Authenticating user root from 192.168.80.11:49472
E20220315 00:37:48.286811    21 QueryInstance.cpp:131] SyntaxError: syntax error near `-graph ('
I20220315 00:37:54.763105    20 GraphService.cpp:68] Authenticating user root from 192.168.80.11:49476
E20220315 00:37:54.767050    21 QueryInstance.cpp:131] SyntaxError: syntax error near `-graphs '
I20220315 00:38:10.096632    20 GraphService.cpp:68] Authenticating user root from 192.168.80.11:49480
I20220315 00:38:15.960521    35 MetaClient.cpp:3045] Load leader of "storaged0":9779 in 0 space
I20220315 00:38:15.960618    35 MetaClient.cpp:3045] Load leader of "storaged1":9779 in 0 space
I20220315 00:38:15.960646    35 MetaClient.cpp:3045] Load leader of "storaged2":9779 in 0 space
I20220315 00:38:15.960685    35 MetaClient.cpp:3051] Load leader ok
I20220315 00:38:31.688366    20 GraphService.cpp:68] Authenticating user root from 192.168.80.11:49486
I20220315 00:38:35.997733    35 MetaClient.cpp:3045] Load leader of "storaged0":9779 in 1 space
I20220315 00:38:35.997900    35 MetaClient.cpp:3045] Load leader of "storaged1":9779 in 1 space
I20220315 00:38:35.998047    35 MetaClient.cpp:3045] Load leader of "storaged2":9779 in 1 space
I20220315 00:38:35.998090    35 MetaClient.cpp:3051] Load leader ok
I20220315 00:39:36.321261    19 GraphService.cpp:68] Authenticating user root from 192.168.80.11:49490
I20220315 00:40:27.838493    19 SwitchSpaceExecutor.cpp:40] Graph switched to `phoros_graph', space id: 1
I20220315 00:58:07.389143    34 ThriftClientManager-inl.h:67] resolve "metad2":9559 as "192.168.80.2":9559
I20220315 00:58:08.394723    34 ThriftClientManager-inl.h:67] resolve "metad2":9559 as "192.168.80.2":9559
E20220315 00:58:09.411571    34 ThriftClientManager-inl.h:70] Failed to resolve address for 'metad0': Name or service not known (error=-2): Unknown error -2
E20220315 00:58:10.417052    34 ThriftClientManager-inl.h:70] Failed to resolve address for 'metad1': Name or service not known (error=-2): Unknown error -2
E20220315 00:58:10.417518    34 MetaClient.cpp:732] Send request to "metad1":9559, exceed retry limit
E20220315 00:58:10.417651    34 MetaClient.cpp:733] RpcResponse exception: apache::thrift::transport::TTransportException: Connection not open: apache::thrift::transport::TTransportException: AsyncSocketException: setReadCallback() called with socket in invalid state, type = Socket not open
E20220315 00:58:10.417804    35 MetaClient.cpp:168] Heartbeat failed, status:RPC failure in MetaClient: apache::thrift::transport::TTransportException: Connection not open: apache::thrift::transport::TTransportException: AsyncSocketException: setReadCallback() called with socket in invalid state, type = Socket not open
E20220315 00:58:20.427565    31 ThriftClientManager-inl.h:70] Failed to resolve address for 'metad1': Name or service not known (error=-2): Unknown error -2
E20220315 00:58:21.431627    31 ThriftClientManager-inl.h:70] Failed to resolve address for 'metad0': Name or service not known (error=-2): Unknown error -2
E20220315 00:58:22.437110    31 ThriftClientManager-inl.h:70] Failed to resolve address for 'metad2': Name or service not known (error=-2): Unknown error -2
E20220315 00:58:23.444401    31 ThriftClientManager-inl.h:70] Failed to resolve address for 'metad1': Name or service not known (error=-2): Unknown error -2
E20220315 00:58:23.444901    31 MetaClient.cpp:732] Send request to "metad1":9559, exceed retry limit
E20220315 00:58:23.445014    31 MetaClient.cpp:733] RpcResponse exception: apache::thrift::transport::TTransportException: Connection not open: apache::thrift::transport::TTransportException: AsyncSocketException: setReadCallback() called with socket in invalid state, type = Socket not open
E20220315 00:58:23.445158    35 MetaClient.cpp:168] Heartbeat failed, status:RPC failure in MetaClient: apache::thrift::transport::TTransportException: Connection not open: apache::thrift::transport::TTransportException: AsyncSocketException: setReadCallback() called with socket in invalid state, type = Socket not open
E20220315 00:58:33.459565    32 ThriftClientManager-inl.h:70] Failed to resolve address for 'metad1': Name or service not known (error=-2): Unknown error -2
E20220315 00:58:34.462333    32 ThriftClientManager-inl.h:70] Failed to resolve address for 'metad2': Name or service not known (error=-2): Unknown error -2
E20220315 00:58:35.468302    32 ThriftClientManager-inl.h:70] Failed to resolve address for 'metad2': Name or service not known (error=-2): Unknown error -2
E20220315 00:58:36.477334    32 ThriftClientManager-inl.h:70] Failed to resolve address for 'metad1': Name or service not known (error=-2): Unknown error -2
E20220315 00:58:36.477845    32 MetaClient.cpp:732] Send request to "metad1":9559, exceed retry limit
E20220315 00:58:36.477962    32 MetaClient.cpp:733] RpcResponse exception: apache::thrift::transport::TTransportException: Connection not open: apache::thrift::transport::TTransportException: AsyncSocketException: setReadCallback() called with socket in invalid state, type = Socket not open
E20220315 00:58:36.478132    35 MetaClient.cpp:168] Heartbeat failed, status:RPC failure in MetaClient: apache::thrift::transport::TTransportException: Connection not open: apache::thrift::transport::TTransportException: AsyncSocketException: setReadCallback() called with socket in invalid state, type = Socket not open
